README.md

```diff
@@ -60,105 +74,279 @@ index.upsert(
 ```python
 from upstash_vector import Vector
 
+# as Vector objects
+
 index.upsert(
     vectors=[
-        Vector(id="id5", vector=[1, 2], metadata={"metadata_f": "metadata_v"}),
+        Vector(id="id5", vector=[1, 2], metadata={"field": "value"}),
         Vector(id="id6", vector=[6, 7]),
     ]
 )
 ```
 
-If you are using an Upstash Vector with an embedding model, you can directly insert data as a string:
+If the index is created with an embedding model, raw string data can be upserted.
 
 ```python
 from upstash_vector import Data
 
 res = index.upsert(
     vectors=[
-        Data(id="id5", data="Goodbye-World", metadata={"metadata_f": "metadata_v"}),
-        Data(id="id6", data="Hello-World"),
+        Data(id="id5", data="Goodbye World", metadata={"field": "value"}),
+        Data(id="id6", data="Hello World"),
     ]
 )
 ```
 
-### Query Index
+Also, a namespace can be specified to upsert vectors into it.
+When no namespace is provided, the default namespace is used.
+
 ```python
-query_vector = [0.6, 0.9]
-top_k = 6
-query_res = index.query(
-    vector=query_vector,
-    top_k=top_k,
-    include_vectors=True,
+index.upsert(
+    vectors=[
+        ("id1", [0.1, 0.2]),
+        ("id2", [0.3,0.4]),
+    ],
+    namespace="ns",
+)
+```
+
+### Query Vectors
+
+Some number of vectors that are approximately most similar to a given
+query vector can be requested from a namespace of an index.
+
+```python
+res = index.query(
+    vector=[0.6, 0.9], 
+    top_k=5,
+    include_vectors=False,
```

Comment:
we print the vector below, we could keep this as `include_vectors=True` so that the code sample works

Reply:
The print below should work. The comment on that line says that the value will be there if it is requested.

```
        r.vector, # The value of the vector, if requested.
```

I converted almost all `include_vectors` to `False` in docs and docstrings because most of the time, users should not need to fetch vectors, and they are very big in terms of size which makes the snippets hard to read when run.

Reply:
got it 👍